### ISSUE TITLE
Task 4 - Notes should be displayed beautifully

### ISSUE THREAD

**Issue by svb-hikip** [svb-hikip/hikipfe-react#30]
![image](https://github.com/svb-hikip/hikipfe-react/assets/172111960/64c9308e-0571-4887-bea9-f4cf074fdbf4)

These client notes are the most important feature. Doctors use it to keep track of the patient. Display all notes beautifully

Use tailwindUI https://tailwindui.com/

Telegram me for the credentials if required

**Comment by AmaamRizvi** [comment]
i want to work on this @svb-hikip

**Comment by svb-hikip** [comment]
I got the API ready. Pull the new changes before you proceed.

API ENDPOINTS AND DOCS
https://hikip-django-e06ccc5f06d1.herokuapp.com/api/schema/swagger-ui/

https://hikip-django-e06ccc5f06d1.herokuapp.com/api/schema/redoc/

**Comment by svb-hikip** [comment]
@AmaamRizvi Please go ahead

**Comment by aditya28n99** [comment]
I noticed that Task 4 was assigned a couple of days ago, but there hasn't been any update since. I am interested in working on this task and would like to contribute. If Task 4 is still available, @svb-hikip, @AmaamRizvi could you please assign it to me or provide any updates regarding its status?

**Comment by svb-hikip** [comment]
@aditya28n99  Pleas go ahead my friend. I want to see both your work. @AmaamRizvi cana lso work on it. I will check both and we shall add best of both worlds to the project.

**Comment by aditya28n99** [comment]
On it!

**Comment by AmaamRizvi** [comment]
> @AmaamRizvi Please go ahead

Okay

**Comment by svb-hikip** [comment]
> I encountered a 404 error when trying to access a ProgressNote instance with a valid UUID (80eea1e0-dfe9-44c0-ae1b-a75110d294cb) via Swagger UI. The request details are as follows: ![404 Not found](https://private-user-images.githubusercontent.com/125676395/339128720-435fd8f6-ac3b-43a5-a353-a0edb93125c2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyNjQwMjYsIm5iZiI6MTcxODI2MzcyNiwicGF0aCI6Ii8xMjU2NzYzOTUvMzM5MTI4NzIwLTQzNWZkOGY2LWFjM2ItNDNhNS1hMzUzLWEwZWRiOTMxMjVjMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxM1QwNzI4NDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNDBlOWEyZGEzMWQzYjBkYjM4NzM2YzU4YzIwYzM4M2M5Zjc0NWUwNDRkMzYxZWIwNTM1MWZlMmVjM2JiNjVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.v7x_GENGGLmWZS7zXEX7GHhLvQiPPx1LjhZqrpdHUFo)

Is it the correct UUID?

**Comment by aditya28n99** [comment]
Yes, UUID is verified as valid in the database.. even i tried different id's. still i got the same error.

**Comment by svb-hikip** [comment]
@aditya28n99  I will check this out, why dont you use the debug variable displaying the payload to make the notes for now?
